Source/SwiftLintFramework/Rules/ConditionalReturnsOnNewline.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  ConditionalReturnsOnNewline.swift
+//  SwiftLint
+//
+//  Created by Rohan Dhaimade on 12/08/2016.
+//  Copyright © 2016 Realm. All rights reserved.
+//
+
+import Foundation
+import SourceKittenFramework
+
+public struct ConditionalReturnsOnNewline: Rule, OptInRule {
```

Comment:
Sounds good. I'm planning on adding more stuff. Could I get some background on what this does and why it's necessary? I figured I didn't need it as I don't have any configurations.